packages/shared/src/components/tooltips/RecommendedMentionTooltip.tsx

```diff
@@ -39,7 +39,7 @@ export function RecommendedMentionTooltip({
       appendTo={document?.body || 'parent'}
       content={
         <RecommendedMention
-          users={query !== undefined && mentions}
+          users={typeof query !== 'undefined' && mentions}
```

Comment:
I think I saw more places where you checked for `undefined` like this `query !== undefined`.
Can you please validate?